src/response.rs

```diff
@@ -282,7 +287,12 @@ impl Response {
     /// # }
     /// ```
     pub fn into_reader(self) -> Box<dyn Read + Send + Sync + 'static> {
-        self.reader
+        match self.reader {
+            PendingReader::Reader(reader) => reader,
+            _ => panic!(
```

Comment:
I have seen a rare case where this occurs, have not figured out why yet. Needs to be fixed.